snuba/datasets/events_processor_base.py

```diff
@@ -246,6 +245,7 @@ def process_insert(
         if metadata is not None:
             processed["offset"] = metadata.offset
             processed["partition"] = metadata.partition
+            processed["message_timestamp"] = metadata.timestamp
```

Comment:
I'd be curious if `processed.update({…})` would be faster here but I didn't want to make this change without benchmarking it.